Makefile

```diff
@@ -219,7 +219,7 @@ install-tools:
 
 .PHONY: run
 run:
-	GO111MODULE=on $(GOCMD) run --race ./cmd/otelcontribcol/... --config ${RUN_CONFIG} ${RUN_ARGS}
+	cd ./cmd/otelcontribcol && GO111MODULE=on $(GOCMD) run --race . --config ${RUN_CONFIG} ${RUN_ARGS}
```

Comment:
Default `RUN_CONFIG?=local/config.yaml` probably won't work with this?

Reply:
done.